src/System.Text.Json/src/System/Text/Json/JsonConstants.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json
+{
+    internal static class JsonConstants
+    {
+        public const byte OpenBrace = (byte)'{';
+        public const byte CloseBrace = (byte)'}';
+        public const byte OpenBracket = (byte)'[';
+        public const byte CloseBracket = (byte)']';
+        public const byte Space = (byte)' ';
+        public const byte CarriageReturn = (byte)'\r';
+        public const byte LineFeed = (byte)'\n';
+        public const byte Tab = (byte)'\t';
+        public const byte ListSeperator = (byte)',';
+        public const byte KeyValueSeperator = (byte)':';
+        public const byte Quote = (byte)'"';
+        public const byte ReverseSolidus = (byte)'\\';
+        public const byte Solidus = (byte)'/';
+        public const byte BackSpace = (byte)'\b';
+        public const byte FormFeed = (byte)'\f';
+        public const byte Asterisk = (byte)'*';
+
+        public static ReadOnlySpan<byte> TrueValue => new byte[] { (byte)'t', (byte)'r', (byte)'u', (byte)'e' };
+        public static ReadOnlySpan<byte> FalseValue => new byte[] { (byte)'f', (byte)'a', (byte)'l', (byte)'s', (byte)'e' };
+        public static ReadOnlySpan<byte> NullValue => new byte[] { (byte)'n', (byte)'u', (byte)'l', (byte)'l' };
```

Comment:
Why are we allocating new arrays every time these are accessed?

Reply:
Isn't there a new feature in C# 7.3, where these properties don't allocate new arrays? I thought defining them this way was better for the first access.

cc @jaredpar

Reply:
Ah, you're right, I forgot that made it in, e.g.
```C#
private static ReadOnlySpan<byte> Bytes => new byte[] { 1, 2, 3 };
```
generates
```
  IL_0000:  ldsflda    valuetype '<PrivateImplementationDetails>'/'__StaticArrayInitTypeSize=3' '<PrivateImplementationDetails>'::'7037807198C22A7D2B0807371D763779A84FDFCF'
  IL_0005:  ldc.i4.3
  IL_0006:  newobj     instance void valuetype [System.Runtime]System.ReadOnlySpan`1<uint8>::.ctor(void*, int32)
  IL_000b:  ret
```

Reply:
I was trying to find the name for this feature or an issue in the roslyn/csharplang repos to get more details, but came up short. Does anyone know?

Reply:
Here's the PR for it: https://github.com/dotnet/roslyn/pull/24621